.env.example

```
@@ -0,0 +1,3 @@
+   SESSION_SECRET=your-secure-secret
```

dotenv-linter (4.0.0)

[warning] [LeadingCharacter] Invalid leading character detected

(LeadingCharacter)